packages/wallets/src/features/cfd/screens/TradingPlatformStatus/TradingPlatformStatus.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { WalletButton, WalletText } from '../../../../components/Base';
+import { useModal } from '../../../../components/ModalProvider';
+import useDevice from '../../../../hooks/useDevice';
```

Comment:
any new `Button`, `Text` or `useDevice` components or usage should be using @deriv-com/ui as part of the current refactoring

Reply:
noted on this, thanks